File: code-coaches/easy/gotham-city/gotham_city.rb
def is_backup_needed(criminals)
    if criminals < 5
        return "I got this!"
    elsif criminals.between?(5,10)
        return "Help me Batman"
    elsif criminals > 10
        return "Good Luck out there!"
    end
end


def main
    criminals = gets.chomp.to_i
    puts is_backup_needed(criminals)
end


if __FILE__ == $0
  main
end

File: code-coaches/easy/gotham-city/gotham_city.cpp
#include <iostream>

std::string is_backup_needed(int criminals)
{
    if (criminals < 5)
		return "I got this!";
	else if (5 <= criminals && criminals <= 10)
		return "Help me Batman";
	else
		return "Good Luck out there!";
}

int main()
{
	int criminals;
	std::cin >> criminals;

	std::cout << is_backup_needed(criminals);

	return 0;
}